frontend/AgriMarketplace.css

```
@@ -441,6 +441,371 @@
+
+.equipment-card {
+  background: rgba(255,255,255,0.92);
+  backdrop-filter: blur(10px);
+  border-radius: 22px;
+  overflow: hidden;
+  border: 1px solid rgba(255,255,255,0.5);
+  box-shadow:
+    0 10px 25px rgba(0,0,0,0.08),
+    0 2px 10px rgba(0,0,0,0.04);
+
+  display: flex;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)